src/modules/users/users.service.ts

```diff
@@ -0,0 +1,12 @@
+import { Injectable } from '@nestjs/common';
+import { CreateUserRequest, UserResponse } from 'src/common/dto';
+import { UsersRepository } from 'src/repository/repositories';
+
+@Injectable()
+export class UsersService {
+	constructor(private readonly usersRepository: UsersRepository) {}
+
+	async createUser(request: CreateUserRequest): Promise<UserResponse> {
+		return this.usersRepository.createUser(request);
+	}
```

Comment:
This flow not finished yet. If you include this implementation in this PR pls comlete create user flow(password: hash, role: 'User') or exlude this fn from PR.